install/minikube-installation-guide.md

```diff
@@ -0,0 +1,240 @@
+$ kubectl apply -f runtimeclass_crd.yaml
+```
+
+### Register the runtime
+
+Now register the `kata qemu` runtime with that class:
+
+```sh
+$ cd packaging/kata-deploy/k8s-1.14
+$ kubectl apply -f kata-qemu-runtimeClass.yaml
+runtimeclass.node.k8s.io/kata created
+runtimeclass.node.k8s.io/kata-qemu created
+```
+
+Kata Containers should now be installed and enabled in the Minikube cluster.
+
+## Testing Kata Containers
+
+Launch a container that is defined to run on Kata Containers. The enabling is configured by
+the following lines in the yaml file:
+
+```yaml
+    spec:
+      runtimeClassName: kata-qemu
+```
+
+Perform the following action to launch a Kata Containers based pod:
+
+```sh
+$ cd packaging/kata-deploy/examples
+$ kubectl apply -f test-deploy-kata-qemu.yaml
+```
+
+An Apache PHP pod should have been deployed with the Kata Containers runtime. Wait a few moments
+to check it is running:
+
+```sh
+$ kubectl get pods
+NAME                                   READY   STATUS    RESTARTS   AGE
+php-apache-kata-qemu-bc4c55994-p6f6k   1/1     Running   0          1m
+```
+
+There are a couple of ways to verify it is running with Kata Containers. Nominally, it
+should be hard to tell - the idea of Kata Containers is that your pod will run inside a VM,
+but look and feel just like it would as a normal software pod. In
+theory, you should not be able to tell your pod is running as a Kata Containers container.
+Careful examination can verify your pod is in fact a Kata Containers pod.
+
+First, look on the node for a `qemu` running, and which version of the kernel the
+node is running:
+
+```sh
+# You should see a VERY long line of qemu here - showing that you are indeed running
+# a qemu VM on the Minikube node - that is the VM that contains your pod.
+$ minikube ssh -- pgrep -a qemu
+
+# For use in a moment, see what kernel is running on the node itself.
+$ minikube ssh -- uname -a
+Linux minikube 4.15.0 #1 SMP Wed Mar 6 23:18:58 UTC 2019 x86_64 GNU/Linux
+```
+
+You should have seen a QEMU process running, indicating that your pod was indeed
+running inside a Kata Containers VM.
+
+Another way to verify Kata Containers is running is to look in the container itself and check
+which  kernel is running there. For a normal software container you will be running
+the same kernel as the node. For a Kata Container you will be running a Kata Containers kernel
+inside the Kata Containers VM.
+
+```sh
+$ podname=$(kubectl get pods -o=name | fgrep php-apache-kata-qemu | sed 's?pod/??')
+$ kubectl exec ${podname} -- uname -a
+Linux php-apache-kata-qemu-7d4647498f-5cl9q 4.19.28 #1 SMP Wed Jun 5 14:32:34 PDT 2019 x86_64 GNU/Linux
+```
+
+There you should see, the node is running kernel 4.15, but the container running under Kata Containers
+is running (in this instance) a 4.19 kernel.
+
+## Wrapping up
+
+This guide should have shown a relatively easy way to get a Minikube up with Kata Containers installed.
```

Comment:
Could this be simplified to the following?:

> This guide has shown an easy way to setup a Minikube with Kata Containers.

Reply:
done